modules/ROOT/pages/_partials/sync/provisioning.adoc

```diff
@@ -0,0 +1,62 @@
+
+It's neccessary to wrap your Sync Server application into Docker container and publish your container to a Docker registry. 
+The easiest way is to use a public one such as link:https://hub.docker.com/[Dockerhub].
+
+= Preparing for deployment
+
+{sync-service} application is a Node.js project, to wrap it into a container, create `Dockerfile` in the project's directory like this:
+
+[source,dockerfile]
+----
+FROM node:8
+WORKDIR /usr/src/app
+# copy Node.js specific files required for Node.js Data Sync application
+COPY package*.json ./
+# copy Data Sync Server app source file to the workdir (add all other files that are needed) 
+COPY server.js . 
+RUN npm install
+# TCP port that Sync Server app is listening on
+EXPOSE 4000
+CMD [ "npm", "start" ]
+---- 
+
+Build the Docker container:
+[source,bash]
+----
+$ docker build .
+----
+
+Push to the Dockerhub repository :
```

Comment:
Thought 1) Maybe we should provide instructions for using the local openshift registry instead of the remote docker hub? 

Thought 2) If we want to include Docker hub then we should probably have a docker hub account in the list of prerequisites for the guide.

Reply:
Let's leave it simple.